root/url/Header.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+// Copyright (C) 2017 MetaBrainz Foundation
+//
+// This file is part of MusicBrainz, the open internet music database,
+// and is licensed under the GPL version 2, or (at your option) any
+// later version: http://www.gnu.org/licenses/gpl-2.0.txt
+
+const React = require('react');
+const {l} = require('../static/scripts/common/i18n');
+const EntityTabs = require('../components/EntityTabs');
+const EntityLink = require('../static/scripts/common/components/EntityLink');
+
+type Props = {
+  url: URLT;
+  page: string;
+  entity_properties: EntityPropsT;
+};
+
+const URLHeader = ({url, page, entity_properties}: Props) => {
+
+  const urlEntity = <EntityLink content={url.decoded} entity={url} subPath={'show'} />;
+
+  let urlObj = urlEntity;
+  if (url.editsPending)
+    urlObj = (
+        <span className="mp">
```

Comment:
Wrapping links in `<span className="mp" />` is already a job performed by the `EntityLink` component: https://github.com/metabrainz/musicbrainz-server/blob/612cc7d/root/static/scripts/common/components/EntityLink.js#L165-L165

I'd remove this and figure out why the existing behavior isn't being triggered.